docs/src/Color.doc.js

```diff
@@ -0,0 +1,61 @@
+// @flow strict
+import type { Node } from 'react';
+import { Flex } from 'gestalt';
+import MainSection from './components/MainSection.js';
+import PageHeader from './components/PageHeader.js';
+import ColorPalette from './components/ColorPalette.js';
+
+const cards: Array<Node> = [];
+const card = (c) => cards.push(c);
+
+card(
+  <PageHeader
+    name="Color"
+    description={`
+    Our color palettes are shared between Brand and Gestalt, and represent our full range of options. For further Brand guidance, please reference the [Brand guidelines website](https://brand.pinterest.com/).
+    `}
+  />,
+);
+
+card(
+  <MainSection
+    name="Full Palettes"
+    description="In order to ensure accessible contrast for color pairings, we require a dark font color to be used for any colors 400 and below. For 500 and above, we recommend using white."
+  >
+    <MainSection.Subsection title="Colors">
```

Comment:
I'm only seeing the first three? This doesn't seem right:
<img width="1367" alt="Screen Shot 2021-08-13 at 2 04 33 PM" src="https://user-images.githubusercontent.com/12059539/129418347-c73fd66f-c9cd-4a25-a93e-b35f1f1da094.png">

Reply:
This should be updated now that the package is published